drivers/wifi/nrf700x/zephyr/src/zephyr_wpa_supp_if.c

```diff
@@ -234,6 +234,10 @@ void wifi_nrf_wpa_supp_event_proc_scan_res(void *if_priv,
 
 	vif_ctx_zep->supp_callbk_fns.scan_res(vif_ctx_zep->supp_drv_if_ctx, r, more_res);
 
+	if (more_res) {
```

Comment:
Shouldn't this be if (!more_res) ?

Reply:
Yes, this is fixed internally and also, we cannot rely on getting scan results in case of no BSSes found, so, preparing an update with more fixes, will push soon. (This is why CI tests were failing).

Reply:
This change need a RPU FW update to always send scan result even for empty, so, on hold for now.

Reply:
Rebased to include the latest patch.